src/layer/place.js

```diff
@@ -394,7 +394,11 @@ export const state = {
     ],
     "text-halo-blur": labelHaloBlur,
   },
-  filter: ["==", ["get", "class"], "state"],
+  filter: [
+    "any",
+    ["==", ["get", "class"], "state"],
+    ["==", ["get", "class"], "province"],
+  ],
```

Comment:
For brevity, use a `match` or `in` expression:

```json
["match", ["get", "class"], ["state", "province"], true, false]
```
```json
["in", ["get", "class"], ["literal", ["state", "province"]]]
```

Reply:
Thank you!  I was originally trying to get an `in` expression to work, but I couldn't quite figure out what the syntax was supposed to be.  Seems I was missing the need to wrap the array in a `literal` expression.